buddyboss/buddyboss-platform
# Fixed - Media uploader -> Select Photos tab more photos does not show photos on select photos tab

### All Submissions:


Fixes #1457 

### How to test the changes in this Pull Request:

1. Go to Profile -> Photos => Any Album Page
2. Click on Add Photos
3. In popup got to "Select Photos" tab
4. click on "Load more" and see photos should load now

### Proof Screenshots or Video
https://prnt.sc/uczfev

### Other information:
